stdlib/public/SwiftShims/SwiftStddef.h

```diff
@@ -13,7 +13,12 @@
 #ifndef SWIFT_STDLIB_SHIMS_SWIFT_STDDEF_H
 #define SWIFT_STDLIB_SHIMS_SWIFT_STDDEF_H
 
+#if !defined(__clang__)
+#include <stdint.h>
+typedef size_t __swift_size_t;
```

Comment:
I'm not convinced that things will be any better here if MSVC is ever modularized, but okay.